### PR TITLE
Add documentation for cakeissues.net

### PR DESCRIPTION
**Pull request by pascalberger**
Add documentation for cakeissues.net, moved from Cake.Issues.Website repository including full history.

Part of cake-contrib/Cake.Issues.Website#123

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    InspectCode(repoRootPath.CombineWithFilePath("MySolution.sln"), settings);
+});
+```
+
+Finally you can define a task where you call the core addin with the desired issue provider.
+
+```csharp
+Task("Analyze-Project")
+.IsDependentOn("Analyze-Project")
+.Does(() =>
```
Comment (x-jokay) [inline docs/examples.md]: Fix indention for `.IsDependentOn` and `.Does`.

File: docs/examples.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    // Read Issues.
+    var issues = ReadIssues(
+        InspectCodeIssuesFromFilePath(logPath),
+        repoRootPath);
+
+    Information("{0} issues are found.", issues.Count());
+});
+```
+
+[JetBrains InspectCode]: https://www.jetbrains.com/help/resharper/InspectCode.html
```
Comment (x-jokay) [inline docs/examples.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/features.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.AffectedFileRelativePath` |                                  |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Line`                     |                                  |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Message`                  |                                  |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Priority`                 |                                  |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.PriorityName`             |                                  |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Rule`                     |                                  |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.RuleUrl`                  | For issues containing a Wiki Url |
+
+[JetBrains InsepectCode]: https://www.jetbrains.com/help/resharper/2017.1/InspectCode.html
+[Cake.Issues.InspectCode addin]: https://www.nuget.org/packages/Cake.Issues.InspectCode
```
Comment (x-jokay) [inline docs/features.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/requirements.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+Order: 10
+Title: Requirements
+Description: Requirements for the Cake.Issues.Inspectcode addin.
+---
+The requirements for using the [Cake.Issues.InspectCode addin] are listed in the [release notes] for any specific version.
+
+[Cake.Issues.InspectCode addin]: https://www.nuget.org/packages/Cake.Issues.InspectCode
+[release notes]: release-notes
```
Comment (x-jokay) [inline docs/requirements.md]: [MD047](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md047---files-should-end-with-a-single-newline-character) markdown files should end with a single newline character.

File: docs/features.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+Description: Features of the Cake.Issues.InspectCode addin.
+---
+The [Cake.Issues.InspectCode addin] provides the following features:
+
+# Basic features
+
+* Reads warnings from [JetBrains InsepectCode] log files.
+* Provides URLs for issues containing a Wiki URL.
+
+# Supported comment formats
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

File: docs/features.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+|                                                                    | Comment format                 | Remarks                               |
+|--------------------------------------------------------------------|--------------------------------|---------------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                       |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                       |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                       |
+
+# Supported IIssue properties
```
Comment (x-jokay) [inline docs/features.md]: [MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.

**Comment by pascalberger** [comment]
@x-jokay Ready for another review.

Top level headings are  required to have the entry appear in site TOC (See https://github.com/cake-contrib/Cake.Issues.Website/pull/41)

**Pull request merged**